app/helpers/application_helper.rb

```diff
@@ -17,14 +17,14 @@ def svg_icon(name, options = {})
     end
   end
 
-  def firm_map_component(center:, adviserPinUrl:, officePinUrl:)
+  def firm_map_component(center:, adviser_pin_url:, office_pin_url:)
```

Comment:
Could you check that these camelcase names are not connected with some Javascript files? My gut is telling me if they are named like this `adviserPinUrl` then it could be also used within JS.

Reply:
Yep - i've verified they're used in a view [here](https://github.com/moneyadviceservice/rad_consumer/blob/master/app/views/firms/partials/_firm_details.html.erb)

Reply:
Sorry, that's a big partial. [Link to the line](https://github.com/moneyadviceservice/rad_consumer/blob/master/app/views/firms/partials/_firm_details.html.erb#L69)

Reply:
:+1: ok yep, I follow now